tests/test_controller.py

```diff
@@ -1120,9 +1120,12 @@ def send(self, *args, **kwargs):
             ])
             response = self.controller.register(do_get=self.http_client.do_get)
 
-        # We get back a ProblemDetail.
+        # We get back a ProblemDetail the first time
+        # we got a problem sending an email. In this case, it was
+        # trying to contact the library's 'help' address included in the
+        # library's authentication document.
         eq_(INTEGRATION_ERROR.uri, response.uri)
-        eq_("SMTP error while sending email to mailto:me@library.org",
+        eq_("SMTP error while sending email to mailto:help@library.org",
```

Comment:
This changed because the controller previously pulled the contact address out of the form submission before running the registration code, so that address (me@library.org) was the first one to fail validation. Now the controller pulls the contact address after running the registration code (this is just to make the code nicer), so the help address (gathered by the registration code) is the first one to fail validation.